src/pages/NewChatPage.js

```diff
@@ -219,42 +241,23 @@ class NewChatPage extends Component {
                             onCloseButtonPress={() => Navigation.dismissModal(true)}
                         />
                         <View style={[styles.flex1, styles.w100, styles.pRelative]}>
-                            {!didScreenTransitionEnd && <FullScreenLoadingIndicator />}
-                            {didScreenTransitionEnd && (
+                            {didScreenTransitionEnd ? (
                                 <OptionsSelector
                                     canSelectMultipleOptions={this.props.isGroupChat}
                                     sections={sections}
                                     selectedOptions={this.state.selectedOptions}
-                                    value={this.state.searchValue}
+                                    value={this.state.searchTerm}
                                     onSelectRow={option => (this.props.isGroupChat ? this.toggleOption(option) : this.createChat(option))}
-                                    onChangeText={(searchValue = '') => {
-                                        const {
-                                            recentReports,
-                                            personalDetails,
-                                            userToInvite,
-                                        } = OptionsListUtils.getNewChatOptions(
-                                            this.props.reports,
-                                            this.props.personalDetails,
-                                            this.props.betas,
-                                            searchValue,
-                                            [],
-                                            this.props.isGroupChat ? this.excludedGroupEmails : [],
-                                        );
-                                        this.setState({
-                                            searchValue,
-                                            userToInvite,
-                                            recentReports,
-                                            personalDetails,
-                                        });
-                                    }}
+                                    onChangeText={this.updateOptionsWithSearchTerm}
                                     headerMessage={headerMessage}
-                                    hideAdditionalOptionStates
-                                    forceTextUnreadStyle
+                                    boldStyle
                                     shouldFocusOnSelectRow={this.props.isGroupChat}
                                     shouldShowConfirmButton={this.props.isGroupChat}
                                     confirmButtonText={this.props.translate('newChatPage.createGroup')}
-                                    onConfirmSelection={this.props.isGroupChat ? this.createGroup : () => {}}
+                                    onConfirmSelection={this.props.isGroupChat && this.createGroup}
```

Comment:
I think this is causing the error I posted about in the thread.
https://github.com/Expensify/App/pull/13319#issuecomment-1338607025